tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -35,18 +33,18 @@ public void run() {
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
-            HttpHandlerMapper httpHandlerMapper = HttpHandlerMapperFactory.create();
+            HttpControllerMapper httpControllerMapper = HttpHandlerMapperFactory.create();
```

Comment:
```suggestion
HttpControllerMapper httpControllerMapper = HttpHandlerControllerFactory.create();
```
이름이 달라서 혼동이 올 수 있을 것 같습니다!

Reply:
와우 체크하지 못했네요.. 감사합니다!!